### ISSUE TITLE
generateJooqClasses is unstable when running on MacOS with Rancher Desktop/Colima

### ISSUE THREAD

**Issue by dmitry-cherkas** [revolut-engineering/jooq-plugin#25]
When running the build on MacOS with Rancher Desktop/Colima we are getting occasional `connection refused` errors, full Gradle log below:

```
16:21:53: Executing 'generateJooqClasses'...

Starting Gradle Daemon...
Gradle Daemon started in 524 ms
127.0.0.1:5432 - no response
waiting for db
127.0.0.1:5432 - accepting connections
> Task :generateJooqClasses FAILED
1 actionable task: 1 executed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generateJooqClasses'.
> 
  Unable to obtain connection from database (jdbc:postgresql://localhost:57409/postgres) for user 'postgres': Connection to localhost:57409 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
  --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  SQL State  : 08001
  Error Code : 0
  Message    : Connection to localhost:57409 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 10s
16:22:05: Execution finished 'generateJooqClasses'.
```
The error is intermittent and is only reproducible with a combination of Rancher Desktop/Colima and MacOS (not observed when running the build on our Linux build agents or when running with Docker Desktop).

I believe that the root cause is distinctly related to this issue: https://github.com/rancher-sandbox/rancher-desktop/issues/2609 which is solved by `.waitingFor(Wait.forListeningPort()))`, so assuming there should be a way to construct a `jooq.image.readinessProbe` in a way that would solve the issue, but I need some advice on how to do that.

Here is our `build.gradle.kts` configuration:
```
tasks.generateJooqClasses {
    schemas = arrayOf("public")
    basePackageName = "com.our.base.package"
    outputDirectory.set(jooqOutputPath)
    excludeFlywayTable = true
    outputSchemaToDefault = setOf("public")

    customizeGenerator {
        name = org.jooq.codegen.KotlinGenerator::class.java.name
    }
}
```

Any advice is greatly appreciated.

**Comment by kamilgregorczyk** [comment]
I'm preparing a fix for it

**Comment by dmitry-cherkas** [comment]
Hi @kamilgregorczyk , this is great news, thank you very much!

**Comment by adrianskrobaczrevolut** [comment]
@dmitry-cherkas the fix has been released in [0.3.9](https://github.com/revolut-engineering/jooq-plugin/releases/tag/0.3.9). Let us know if it works for you

**Comment by dmitry-cherkas** [comment]
Hi @adrianskrobaczrevolut, I've did some testing and the issue is not reproducible with [0.3.9](https://github.com/revolut-engineering/jooq-plugin/releases/tag/0.3.9). Thanks a lot for taking time to fix it!
